src/rust/inetstack/protocols/layer4/tcp/established/receiver.rs

```diff
@@ -35,7 +35,7 @@ use ::futures::never::Never;
 // TODO: Review this value (and its purpose).  It (16 segments) seems awfully small (would make fast retransmit less
 // useful), and this mechanism isn't the best way to protect ourselves against deliberate out-of-order segment attacks.
 // Ideally, we'd limit out-of-order data to that which (along with the unread data) will fit in the receive window.
-const MAX_OUT_OF_ORDER_SIZE_FRAMES: usize = 16;
+const MAX_OUT_OF_ORDER_SIZE_FRAMES: usize = 1024;
```

Comment:
That's a big jump. I'm wondering if we should add a test somehow to exercise this case?

Reply:
We can add a trace but it'll be very long because we need to send 1024 frames out of order. I don't think we need a test because this number is an upper bound and will only be allocated if we have a long sequence of out of order frames.